docs/docs/aleph_zero/collections.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Collections
+
+slug: /aleph_zero/collecions
+---
+
+## Positions
+
+```rust
+#[ink::storage_item]
+#[derive(Debug, Default)]
+pub struct Positions {
+    positions: Mapping<AccountId, (u32, Vec<Position>)>,
+}
+```
+
+|Key|Value|
+|-|-|
+|Unique account identifier| A tuple containing the number of positions (u32) and a vector of `Position` objects |
+
+The `Positions` struct is designed to manage positions associated with different accounts. It uses a mapping data structure, where each account is uniquely identified by its `AccountId`, and a tuple containing the number of positions and a vector of positions is stored as the associated value. The provided functions allow you to add, update, remove, transfer, and retrieve positions associated with specific accounts.
+
+This documentation should help users understand the purpose of the `Positions` struct and how to use its functions effectively.
+
+**Why Positions are Stored in State Instead of NFTs:**
+
+We've chosen to store positions in the state rather than using NFTs for several reasons:
+
+1. **Iterating Through NFTs**: Iterating through NFTs can be problematic and less efficient, especially when dealing with a large number of unique NFTs. Storing positions in a state makes it easier to manage and access them.
```

Comment:
Merge 1 and 2. Optimal determining user's positions and iterating through them.
Main reason: independence from blockchain token indexer